src/entities/trades/curve/tokens/tokens.ts

```diff
@@ -0,0 +1,592 @@
+    symbol: 'stETH',
+    decimals: 18,
+    type: TokenType.ETH,
+  },
+  snx: {
+    address: '0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f',
+    name: 'Synthetix Network Token',
+    symbol: 'SNX',
+    decimals: 18,
+    type: TokenType.OTHER,
+  },
+  slink: {
+    address: '0xbBC455cb4F1B9e4bFC4B73970d360c8f032EfEE6',
+    name: 'Synth sLINK',
+    symbol: 'sLINK',
+    decimals: 18,
+    type: TokenType.LINK,
+  },
+  link: {
+    address: '0x514910771AF9Ca656af840dff83E8264EcF986CA',
+    name: 'ChainLink Token',
+    symbol: 'LINK',
+    decimals: 18,
+    type: TokenType.LINK,
+  },
+  seur: {
+    address: '0xD71eCFF9342A5Ced620049e616c5035F1dB98620',
+    name: 'Synth sEUR',
+    symbol: 'sEUR',
+    decimals: 18,
+    type: TokenType.EUR,
+  },
+  eurs: {
+    address: '0xdB25f211AB05b1c97D595516F45794528a807ad8',
+    name: 'STASIS EURS Token',
+    symbol: 'EURS',
+    decimals: 2,
+    type: TokenType.EUR,
+  },
+  linkusd: {
+    name: 'LINKUSD',
+    symbol: 'LINKUSD',
+    decimals: 18,
+    address: '0x0E2EC54fC0B509F445631Bf4b91AB8168230C752',
+    type: TokenType.LINK,
+  },
+  spell: {
+    name: 'Spell',
+    symbol: 'SPELL',
+    decimals: 18,
+    address: '0x090185f2135308BaD17527004364eBcC2D37e5F6',
+    type: TokenType.SPELL,
+  },
+  cvx: {
+    name: 'Convex',
+    symbol: 'CVX',
+    decimals: 18,
+    address: '0x4e3FBD56CD56c3e72c1403e103b45Db9da5B9D2B',
+    type: TokenType.CVX,
+  },
+  xaut: {
+    name: 'Tether Gold',
+    symbol: 'XAUt',
+    decimals: 6,
+    address: '0x68749665FF8D2d112Fa859AA293F07A622782F38',
+    type: TokenType.GOLD,
+  },
+  t: {
+    name: 'Threshold Network',
+    symbol: 'T',
+    decimals: 18,
+    address: '0xCdF7028ceAB81fA0C6971208e83fa7872994beE5',
+    type: TokenType.T,
+  },
+  eurt: {
+    name: 'Tether Euro',
+    symbol: 'EURT',
+    decimals: 6,
+    address: '0xC581b735A1688071A1746c968e0798D642EDE491',
+    type: TokenType.EUR,
+  },
+  tricrv: {
+    name: 'Curve.fi DAI/USDC/USDT (3Crv)',
+    symbol: '3Crv',
+    decimals: 18,
+    address: '0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490',
+    isLPToken: true,
+    type: TokenType.USD,
+  },
+  sbtccrv: {
+    symbol: 'sbtcCrv',
+    name: 'Curve.fi renBTC/wBTC/sBTC',
+    decimals: 18,
+    address: '0x075b1bb99792c9E1041bA13afEf80C91a1e70fB3',
+    isLPToken: true,
+    type: TokenType.BTC,
+  },
+  rai: {
+    name: ' Rai Reflex Index',
+    symbol: 'RAI',
+    decimals: 18,
+    address: '0x03ab458634910AaD20eF5f1C8ee96F1D6ac54919',
+    type: TokenType.OTHER,
+  },
+  rsv: {
+    address: '0x196f4727526eA7FB1e17b2071B3d8eAA38486988',
+    name: 'Reserve',
+    symbol: 'RSV',
+    decimals: 18,
+    type: TokenType.OTHER,
+  },
+  crvrenwsbtc: {
+    address: '0x075b1bb99792c9E1041bA13afEf80C91a1e70fB3',
+    name: 'Curve.fi renBTC/wBTC/sBTC',
+    symbol: 'crvRenWSBTC',
+    decimals: 18,
+    type: TokenType.OTHER,
+  },
+}
+
+// type StyleSet<T extends string> = {
+//   [k in T]: CurveToken
+// }
```

Comment:
This can be deleted

Reply:
Addressed in [59c2ccc](https://github.com/levelkdev/swapr-sdk/pull/71/commits/59c2ccc7ad2365df40c53cccdeab3949ccd8a83e)